package/boot/uboot-envtools/files/qualcommax_ipq60xx

```diff
@@ -9,7 +9,8 @@ board=$(board_name)
 
 case "$board" in
 8devices,mango-dvk|\
-8devices,mango-dvk-sfp)
+8devices,mango-dvk-sfp|\
+cambiumnetworks,xe3-4)
```

Comment:
You dont really need this because you added the U-Boot compatible to the environment partition, uboot-envtools will find that automatically.